L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -290,12 +290,16 @@ namespace trklet {
     void setStripLength_2S(double stripLength_2S) { stripLength_2S_ = stripLength_2S; }
 
     //Following functions are used for duplicate removal
-    //Function which gets the value corresponding to the overlap size for the overlap rinv bins in DR
+    //Function which returns the value corresponding to the overlap size for the overlap rinv bins in DR
     double overlapSize() const { return overlapSize_; }
```

Comment:
Given the addition of phi, it'd be good to call this rinvOverlapSize() now.

Reply:
Done.

Reply:
@dally96 your reply says Tova's comment has been implemented, but the code remains unchanged?

Reply:
It's been changed now.